<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActorEditTool</class>
 <widget class="QDialog" name="ActorEditTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="modelledActor">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Modelled actor properties</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Animation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxAnimation"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Particle effect properties</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Edit particle effect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="TransformEdit" name="transform" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonBrushToggle">
       <property name="text">
        <string>Activate brush</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Select actor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>New actor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Delete selected</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TransformEdit</class>
   <extends>QWidget</extends>
   <header>transformedit.h</header>
   <container>1</container>
   <slots>
    <signal>rotationChanged()</signal>
    <signal>scaleChanged()</signal>
    <signal>translationChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ActorEditTool</receiver>
   <slot>onNewActor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxAnimation</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ActorEditTool</receiver>
   <slot>onAnimationSelected(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBrushToggle</sender>
   <signal>clicked()</signal>
   <receiver>ActorEditTool</receiver>
   <slot>onToggleBrush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ActorEditTool</receiver>
   <slot>onSelectActor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transform</sender>
   <signal>rotationChanged()</signal>
   <receiver>ActorEditTool</receiver>
   <slot>onRotationChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transform</sender>
   <signal>scaleChanged()</signal>
   <receiver>ActorEditTool</receiver>
   <slot>onScaleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transform</sender>
   <signal>translationChanged()</signal>
   <receiver>ActorEditTool</receiver>
   <slot>onTranslationChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ActorEditTool</receiver>
   <slot>onDeleteActor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ActorEditTool</receiver>
   <slot>onEditParticleEffect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onNewActor()</slot>
  <slot>onAnimationSelected(QString)</slot>
  <slot>onToggleBrush()</slot>
  <slot>onAddDoodad()</slot>
  <slot>onSave()</slot>
  <slot>onSelectActor()</slot>
  <slot>onScaleChanged()</slot>
  <slot>onTranslationChanged()</slot>
  <slot>onRotationChanged()</slot>
  <slot>onDeleteActor()</slot>
  <slot>onBBoxGeometryChanged()</slot>
  <slot>onEditParticleEffect()</slot>
 </slots>
</ui>
